Source/Core/Core/AchievementManager.cpp

```diff
@@ -812,11 +812,16 @@ void AchievementManager::HandleAchievementChallengeIndicatorHideEvent(
 void AchievementManager::HandleAchievementProgressIndicatorShowEvent(
     const rc_client_event_t* client_event)
 {
-  const auto& instance = AchievementManager::GetInstance();
+  auto& instance = AchievementManager::GetInstance();
+  auto current_time = std::chrono::steady_clock::now();
+  const auto message_wait_time = std::chrono::milliseconds{OSD::Duration::SHORT} * 2;
```

Comment:
You originally had this at 2 seconds.  If you want to keep that, you'd want:

`const auto message_wait_time = std::chrono::milliseconds{OSD::Duration::SHORT};`

Reply:
Though I see the achievement progress indicator is 2 seconds already, so this would at least be long enough.

Reply:
Genuinely don't know what I was thinking by adding the *2 there.